web-console/3-project-details/binding-in-project.md

```diff
@@ -0,0 +1,77 @@
+
+
+# Binding in Context
+
+- Users may bind Provisioned Services to other applications in their project and vice versa.
+- The binding flow will use the [Overlay Panel Pattern](http://openshift.github.io/openshift-origin-design/web-console/4-patterns/overlay-panel) and employ a similar wizard experience to the [ordering experience](http://openshift.github.io/openshift-origin-design/web-console/5-components/order-from-catalog).
+
+
+
+
+## Initiating the bind flow
+
+![template](img/bind_kebab.png)
```

Comment:
Looks like the examples on patternfly use sentence capitalization. Removing the hyphens.